plugins/com.google.cloud.tools.eclipse.appengine.deploy.test/src/com/google/cloud/tools/eclipse/appengine/deploy/flex/FlexWarStagingDelegateTest.java

```diff
@@ -56,6 +56,13 @@ public void testStage() {
     IStatus status = delegate.stage(stagingDirectory, safeWorkDirectory,
         null, null, new NullProgressMonitor());
 
+    // Diagnostic message: https://github.com/GoogleCloudPlatform/google-cloud-eclipse/issues/3107
+    if (!status.isOK()) {
```

Comment:
This should really be part of the failure message

Reply:
Done.